updateProject.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <base target="_top">
+  </head>
+  <body>
+
+  <h2>Coderbunker Timesheet</h2>
+
+  <br>
+  <style>
```

Comment:
keep <style></style> in <head></head>, don't put it in the <body></body> part

Reply:
Sorry, I didn't quite get what you meant - Do you mean by making a separate file only with CSS?